030-Concepts/080-pricing.mdx

```diff
@@ -7,26 +7,195 @@ slug: concepts/pricing
+
+- **High availability (HA):** Consistent service access with minimal interruptions or downtime.
+
+- **Elasticsearch integration:** Integrates efficient and responsive full-text search capabilities within your projects. Our search engine uses Elasticsearch, making searching a breeze because it replicates all the data from our database. You get 2 GB of space here initially, but you can buy more, or go custom, depending on what you need.
+
+- **File attachments:** Xata uses S3 and Cloudflare's CDN for storing and delivering files. Your files are stored safely, and you access them quickly when you need them. Starting storage is 2 GB, but you can add more or customize it as per your needs.
+
+- **Image transformations:** You can do 10k image transformations from the start and add more as needed. 
+
+- **AI questions:** Xata offers an option for integrating with OpenAI and ChatGPT, allowing you to ask up to 250 AI-powered questions. If you want to ask more questions, you can, and yes, custom plans are available here as well.
+
+- **Support:** Depending on your plan, you get different levels of support, starting from community support to dedicated support.
+
+
+### Free plan
+
+The Free plan is perfect for individuals who are working on personal projects or are in the experimentation phase. This plan offers the basic functionalities needed to get started, at no cost.
+
+**The Free plan is ideal for:**
+  - Individual developers
+  - Students
+  - Hobbyists
+  - Experimenters, who want to learn more about Xata
+
+##### Features
+- Database branches: Manage your relational and structural data using up to 10 database branches.
+- Availability: High availability to ensure your projects are always up and running, with minimal disruptions.
+- Data storage: 15 GB to store the data related to your projects.
+- Storage: 2 GB for search engine and file attachments. You can use this space for enabling full-text search and storing file attachments.
+- AI queries/month: Run queries with AI integration up to 250 times a month.
+- Support: Community support is offered, so you can engage with community forums for assistance and discussions.
+
+
+### Pro plan
+
+The Pro Plan is optimized for small to medium-sized teams that require more resources. It operates on a usage-based pricing model, allowing users to scale resources as needed. 
+Each month has a starting charge of $20. If there are any additional costs from extra usage during the month, users will see those charges on their account on the first day of the next month.
+
+**The Pro plan is ideal for:**
+  - Small to medium-sized teams
+  - Startups
+  - Growing projects
+
+> If you're affiliated with a startup or a non-profit organization, contact us. We have specific programs and discounts designed especially for startups and non-profits.
+
+##### Features
```

Comment:
I think it's important to pick ONE source of truth for these lists of features. I'd advocate it's our `/pricing` page since it's going to be the easiest to read (it is a designed table) and the easiest to find (it's in the header after all). I think it's fine for the docs to reference by sayng "for a full list of features and costs, please check our pricing page".

My assumption was we want this page in the docs for people if they have more questions. Like how are metrics calculated. Not necessarily how much they cost.

Don't mean to get meta here, I just want to make sure before doing a review.

Reply:
+1, I just dropped in a similar comment.

Reply:
We don't have dedicated instances yet, but when we do, we should list out the different usage metrics (their definitions and values by instance) or simply link out to aurora.